en/setup/dev_env_linux_boutique.md

```diff
@@ -126,100 +126,7 @@ After that, logging out and logging back in is needed.
 
 > **Note** Log out and log in for changes to take effect! Also remove the device and plug it back in!**
 
-### Toolchain Installation
 
-Execute the script below to either install GCC 4.9 or 5.4:
-
-**GCC 4.9**:
-
-```sh
-pushd .
-cd ~
-wget https://launchpad.net/gcc-arm-embedded/4.9/4.9-2015-q3-update/+download/gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2
-tar -jxf gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar.bz2
-exportline="export PATH=$HOME/gcc-arm-none-eabi-4_9-2015q3/bin:\$PATH"
-if grep -Fxq "$exportline" ~/.bash_profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
-popd
-```
-
-**GCC 5.4**:
-
-```sh
-pushd .
-cd ~
-wget https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2
-tar -jxf gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2
-exportline="export PATH=$HOME/gcc-arm-none-eabi-5_4-2016q2/bin:\$PATH"
-if grep -Fxq "$exportline" ~/.bash_profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
-popd
-```
-
-Now restart your machine.
-
-**If using Debian Linux, run this command:**
-
-<div class="host-code"></div>
-
-```sh
-sudo dpkg --add-architecture i386
-sudo apt-get update
-```
-
-**and Install the 32 bit support libraries** (this might fail and can be skipped if running a 32 bit OS):
-
-<div class="host-code"></div>
-
-```sh
-sudo apt-get install libc6:i386 libgcc1:i386 libstdc++5:i386 libstdc++6:i386
-sudo apt-get install gcc-4.6-base:i386 
-```
-
-## Ninja Build System
-
-Ninja is fast than Make and the PX4 CMake generators support it. Unfortunately Ubuntu carries only a very outdated version at this point. To install a recent version of [Ninja](https://github.com/martine/ninja), download the binary and add it to your path:
-
-<div class="host-code"></div>
-
-```sh
-mkdir -p $HOME/ninja
-cd $HOME/ninja
-wget https://github.com/martine/ninja/releases/download/v1.6.0/ninja-linux.zip
-unzip ninja-linux.zip
-rm ninja-linux.zip
-exportline="export PATH=$HOME/ninja:\$PATH"
-if grep -Fxq "$exportline" ~/.profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
-. ~/.profile
-```
-
-## Troubleshooting
-
-### Version Test
-
-Enter:
-
-<div class="host-code"></div>
-
-```sh
-arm-none-eabi-gcc --version
-```
-
-The output should be something similar to:
-
-<div class="host-code"></div>
-
-```sh
-arm-none-eabi-gcc (GNU Tools for ARM Embedded Processors) 4.7.4 20140401 (release) [ARM/embedded-4_7-branch revision 209195]
-Copyright (C) 2012 Free Software Foundation, Inc.
-This is free software; see the source for copying conditions.  There is NO
-warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-```
-
-If you get:
-
-<div class="host-code"></div>
-
-```sh
-arm-none-eabi-gcc --version
-arm-none-eabi-gcc: No such file or directory
-```
-make sure you have the 32bit libs installed properly as described in the installation steps.
+### GCC Toolchain Installation
```

Comment:
@LorenzMeier What can you build with CentOS/Arch versions? Specifically:
1. Are these GCC instructions supposed to be here too?
2. What about all the other things we have in the Ubuntu version - Ninja build etc. SHould these be here, and are they installed in the same way?
3. What can you build with the instructions - simulator? nuttx? snapdragon? rapi?